using-funds/cash-advances.md

```diff
@@ -0,0 +1,9 @@
+# Cash Advances
+
+After being approved for a [grant](https://github.com/bridgefoundry/operations/tree/master/grants), Independent Communities that can't afford to get reimbursed after the fact can receive funds via a cash advance.
+
+The big downside of a cash advance is that you will need to send back any extra money that you did not spend. If you can afford to do so, it is best to low ball your estimate and then get reimbursed for a the small amount that you go over.
+
+If you opt to receive a cash advance, you must **submit all receipts from the workshop within 7 days of the workshop and must follow all of our receipt policies exactly**, especially including itemization and proof of payment for each expense. Failure to do so or to return unused funds will prevent you from being able to receive another cash advance for future workshops.
+
+If you select the cash advance option on the grant application, we will reach out to you about how to send you the money. We are currently investigating the best options for sending cash advances, but at this time we anticipate check and PayPal being the two primary options. 
```

Comment:
I do like the mechanism that we had been considering where somebody with a pex card simply makes the purchases online.  Most food could be purchased this way these days.

Reply:
I spoke with our accountant about that idea and he confirmed it's not an option for grants, which is the relationship with independent communities. So we can call in with a card for Chapters (see "direct pay" page) but need this cash advance option for Communities that can't afford to get reimbursed.

Reply:
:(